### DISCUSSION TITLE
Backup problem

### DISCUSSION THREAD

**Discussion by pusherbox** [immich-app/immich#2371] [Q&A]
Hi all,
My library has 3131 pics and videos, but my immich backup status says this: 
![image](https://user-images.githubusercontent.com/28348268/235704704-128a6c71-1245-4d9a-952b-5bd736b51e22.jpeg)
During the first upload it has freezes a couple of times and I had to force close the app and launch it again...
How can I fix this?
If I take a new picture immich backs it up with no problem but I would like to fix it... have I made something wrong?
Thank you in advance

**Comment by samip5** [comment]
Known visibility bug

**Reply by pusherbox** [reply]
Oh I see... is it still an open bug? I tried to find something with no success. Do you have any pins to follow and to keep updated? Thank you in advance

**Reply by samip5** [reply]
If you search for negative, you will find the bug report(s). https://github.com/immich-app/immich/issues/968

**Reply by pusherbox** [reply]
Good. Thank you
